data/sources/supporting-zoning.json

```diff
@@ -7,6 +7,16 @@
       "sql": "SELECT cartodb_id AS id, * FROM dcp_commercial_overlays",
       "dataPipeline": true
     },
+    {
+      "id": "zoning-map-index",
+      "sql": "SELECT cartodb_id AS id, CASE WHEN sectionalm ILIKE '%Subplan' THEN sectionalm ELSE dzm_sect_n END AS zmi_label , * FROM dcp_zoning_map_index",
```

Comment:
@TylerMatteo  I had to duplicate this `CASE` block here on line 12 to get the tool tips working on hover and on line 17 to place the ZMI label in the middle of the section